package-lock.json

```diff
@@ -1501,6 +1500,25 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/tmp/node_modules/glob": {
```

Comment:
💡 Please note: since the `package.json` has no new packages update or version change beside the extension version bump might be run `rpm install` to make sure this is not intentional change.

Reply:
yup this got generated from npm install. I re-run npm install and got deleted, so not entirely sure how it was added to begin with.